deploy/complete/kubernetes/mushop.yaml

```diff
@@ -300,7 +291,7 @@ metadata:
     app.kubernetes.io/version: "0.1.0"
     app.kubernetes.io/managed-by: Helm
 data:
-  atp.init.sql: ""
```

Comment:
can we leave this empty for this file?

Reply:
Where you are seeing this?

Reply:
You are asking to keep empty or asking why is empty?

Reply:
saying it should stay empty... if possible